landoui/templates/partials/navbar.html

```diff
@@ -1,34 +1,83 @@
-<nav class="Navbar nav">
-  <div class="nav-left">
-    <a class="nav-item" href="/">
+<nav class="Navbar navbar">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="/">
       <img
         class="Navbar-logo"
         src="{{ url_for('static', filename='images/logo/bird_128.png') }}"/>
       <span class="Navbar-title">LANDO</span>
       <span class="Navbar-subtitle">@MOZILLA</span>
     </a>
+
+    <div class="navbar-burger">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
   </div>
 
-  <div class="nav-right">
-    <div class="nav-item">
-      <div class="field is-grouped">
-        <p class="control">
-          <a class="Navbar-login button">
-            <span class="icon">
-              <i class="fa fa-user-circle"></i>
-            </span>
-            <span>Login with Auth0</span>
-          </a>
-        </p>
-        <p class="control">
-          <a class="button" >
-            <span class="icon">
-              <i class="fa fa-question-circle"></i>
-            </span>
-            <span>Help</span>
-          </a>
-        </p>
+  <div id="Navbar-menu" class="navbar-menu">
+    <div class="navbar-end">
+      <div class="navbar-item">
+        <div class="field is-grouped">
+          {% if 'id_token' in session %}
+            <p class="control">
+              <a class="Navbar-userSettingsBtn button">
+                <span class="icon">
+                  <img src="{{ session['userinfo']['picture'] }}" />
+                </span>
+                <span>&nbsp;{{ session['userinfo']['name'] }}&nbsp;</span>
+                <i class="fa fa-cogs"></i>
+              </a>
+            </p>
+          {% endif %}
+          <p class="control">
+            {% if 'id_token' in session %}
+              <a class="button" href="/logout">
+                <span class="icon">
+                  <i class="fa fa-sign-out"></i>
+                </span>
+                <span>Logout</span>
+              </a>
+            {% else %}
+              <a class="Navbar-login button" href="/signin">
+                <span class="icon">
+                  <i class="fa fa-sign-in"></i>
+                </span>
+                <span>Login with Auth0</span>
+              </a>
+            {% endif %}
+          </p>
+          <p class="control">
+            <a class="button" >
+              <span class="icon">
+                <i class="fa fa-question-circle"></i>
+              </span>
+              <span>Help</span>
+            </a>
+          </p>
+        </div>
       </div>
     </div>
   </div>
+
+  <div class="Navbar-modal modal">
```

Comment:
We'll want to do a `a11y` sweep before launch.  I think we should have a good look at this post about how we can make our modals easier to use for everyone.  This is a good read:

https://www.smashingmagazine.com/2014/09/making-modal-windows-better-for-everyone/